src/nvim/ex_getln.c

```diff
@@ -4061,7 +4061,7 @@ static int showmatches(expand_T *xp, int wildmenu)
   if (got_int)
     got_int = FALSE;            /* only int. the completion, not the cmd line */
   else if (wildmenu)
-    win_redr_status_matches(xp, num_files, files_found, 0, showtail);
+    win_redr_status_matches(xp, num_files, files_found, -1, showtail);
   else {
```

Comment:
You will probably see the [lint job](https://travis-ci.org/neovim/neovim/builds/334501761) fail, because Nvim tries to improve the codebase of Vim step-by-step. That involves style chances as well. Nvim requires to always use braces when using statements like `if`/`else if`/`else`.

Our CI checks for the required style by looking at the surrounding lines of changed lines in a commit. So.. you will probably need to change this to:

```vim
if (got_int) {
  got_int = FALSE;    // only int. the completion, not the cmd line
} else if (wildmenu) {
  win_redr_status_matches(xp, num_files, files_found, -1, showtail);
} else {
```

Reply:
done

Reply:
Try running `make clint`. It complains about a few other things as well.

(I know this can be annoying if you just want to fix the code, but unfortunately it's needed with Vim's old codebase.)

Reply:
I executed `make clint` and fixed the linking errors. Sry, my first PR here.

Reply:
No need to be sorry. ;-) Well done!

I'll just wait for all tests to be green and then merge it afterwards.